module/Application/language/en.po

```diff
@@ -976,6 +991,16 @@ msgstr "Query Result"
 msgid "Read organ membership (per user/organ)"
 msgstr "Read organ membership (per user/organ)"
 
+msgid "Reappoint"
+msgstr ""
```

Comment:
If you do this, some strings will disappear throughout the code; when the `msgid` is not present, it will simply list the msgid, but once it is in there, it will display the empty string as translation

Reply:
Or you can exclude the language files from this update, and it can be done separately; usually @tomudding prefers a separate translation merge, although it is less of an issue now the files are sorted